api/src/test/scala/uk/ac/wellcome/platform/api/services/ElasticsearchServiceTest.scala

```diff
@@ -46,41 +46,86 @@ class ElasticsearchServiceTest
     // for different sort orders.
   }
 
-  it("should return the correct number of results from Elasticsearch") {
-    val work1 = identifiedWorkWith(
-      canonicalId = "0001",
-      label = "The first flounder"
-    )
-    val work2 = identifiedWorkWith(
-      canonicalId = "0002",
-      label = "The second salmon"
-    )
-    val work3 = identifiedWorkWith(
-      canonicalId = "0003",
-      label = "The third trout"
+  /** Populate the test index with some works, and return the corresponding
+   *  `DisplayWork` instances.
+   */
+  private def populateElasticsearch(count: Int = 10): List[DisplayWork] = {
+    val works: List[IdentifiedWork] = (0 to 9).map { x =>
+      identifiedWorkWith(canonicalId = s"ID-$x", label=s"Work number $x")
+    }.toList
+    insertIntoElasticSearch(works: _*)
+
+    works.map { work: IdentifiedWork =>
+      DisplayWork("Work", work.canonicalId, work.work.label)
```

Comment:
Convenience, mainly. That’s what the tests already had. I agree that these should be agnostic about the type of object they’re dealing with, but we have `Work` running throughout this setup.

I’d have no objection to pulling out the dependency on Work at a later date, but I don't fancy unpicking all those assumptions right now.